tests/function-controller/testsuite/testsuite.go

```diff
@@ -136,6 +136,10 @@ func (t *TestSuite) Run() {
 	ns, err := t.namespace.Create()
 	failOnError(t.g, err)
 
+	t.t.Log("Creating function without body ready to rejection by the webhook")
+	err = t.function.Create(&functionData{})
+	t.g.Expect(err).NotTo(gomega.BeNil())
```

Comment:
Can we somehow also test mutating webhook in those tests? Like in an original function in those tests we could not provide anything about min/max replicas, and then after the function is created those fields should be defaulted. If there would be no webhooks then there would not even be such a field, as it is declared as `*int32`(spec.MinReplicas would be nil)